css/style.css

```
@@ -6,14 +6,165 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+header img{
+    width:50px;
+    height:50px;
+}
+header{
+    display:flex;
+    flex-direction: row;
```

```suggestion
    align-items: center;
    padding: 1rem;
```
`flex-direction` is `row` by default so we don't have to add it.

Also, we can add some padding to the header to add more space around the edges, and then use `align-items: center` to centre the logo and links vertically